validation/validate.go

```diff
@@ -66,7 +68,7 @@ type SpecValidationError struct {
 }
 
 func (s StepValidationError) Error() string {
-	return fmt.Sprintf("%s:%d %s => '%s'", s.fileName, s.step.LineNo, s.message, s.step.GetLineText())
+	return fmt.Sprintf("%s:%d %s => '%s'%s", s.fileName, s.step.LineNo, s.message, s.step.GetLineText(), s.quickFix)
```

Comment:
Check with @zabil if the quick fixes need to be inline with the error, as against a summary at the end of the run